wiki/Beatmap_information/en.md

```diff
@@ -0,0 +1,141 @@
+
+Underneath the preview menu the song length, the BPM, as well as the number of [hit objects](/wiki/Gameplay/Hit_object) can be seen from left to right respectively. Upon hovering over the song length, a tooltip will additionally show the [drain](/wiki/Beatmap/Drain_time) length.
+
+Depending on the game mode, the following beatmap settings may be visible, along with the corresponding value next to it:
+
+| Setting | Description | Game mode |
+| :-: | :-- | :-: |
+| [Circle size](/wiki/Beatmap/Circle_size) (CS) | Determines how big the hit circles are. | ![][osu!] ![][osu!catch] |
+| [HP drain](/wiki/Beatmap/HP_drain_rate) (HP) | Defines how much health is gained and lost while playing. | ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania] |
+| [Approach rate](/wiki/Beatmap/Approach_rate) (AR) | Regulates the speed hit objects appear in. | ![][osu!] ![][osu!catch] |
+| [Accuracy](/wiki/Gameplay/Accuracy) | Measures how good the player hits hit objects on time. | ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania] |
+| Key count | Specifies the number of keys used when playing a beatmap. | ![][osu!mania] |
+
+At the bottom of the statistics panel is the [star rating](/wiki/Beatmap/Star_rating) detailing how much skill the player requires for the beatmap.
+
+::: Infobox
+![](img/user-rating.png "User rating")
+:::
+
+If the beatmap is [qualified](/wiki/Beatmap/Category#qualified), [ranked](/wiki/Beatmap/Category#ranked), or [loved](/wiki/Beatmap/Category#loved), a user rating will be displayed below the statistics panel. User rating is a feature in osu!(stable) where after passing a beatmap users are able to vote between 1 and 10 stars based on how much they liked it.
+
+If the user votes 6 stars or more (positive vote), the bar will be coloured green, otherwise yellow (negative vote). The numbers to the side of the bar describe the number of users that have voted either positively or negatively. Above the bar is a number in brackets displaying the average number of stars every user voted for.
+
+The rating spread below the user rating illustrates the proportionate amount of user votes for each number of stars.
+
+## Metadata
+
+### Beatmap description
+
+The beatmap description is a custom field often used for:
+
+- linking ressources such as the background image or hitsound samples
+- commending other users for their help (e.g. guest mappers, modder, storyboarder)
+- trivia facts related to the beatmap (e.g. mapping milestones)
+
+### Keywords
+
+In addition to gameplay-related statistics, every beatmap includes metadata fields to help categorise the beatmap's song. Before submitting a beatmap as part of the [ranking process](/wiki/Beatmap_ranking_procedure), a mapper is required to add this information to the beatmap.
+
+- [Genre](/wiki/Beatmap/Genre_and_language#list-of-genres): The musical or media genre this song mainly uses.
```

Comment:
Weird way to explain it. It's not the genre of the media where it comes from, but rather the song itself.
```suggestion
- [Genre](/wiki/Beatmap/Genre_and_language#list-of-genres): The song's main musical genre.
```